bundles/org.openhab.binding.solax/pom.xml

```diff
@@ -14,4 +14,12 @@
 
   <name>openHAB Add-ons :: Bundles :: Solax Binding</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>javax.measure</groupId>
+      <artifactId>unit-api</artifactId>
+      <version>1.0</version>
+    </dependency>
+  </dependencies>
```

Comment:
Where is the usage of this? I might have missed it as I only browsed files in GitHub.

Reply:
I believe I added this one for backwards compatibility with a 4.0 version of OH. There was some issue with initialization of the binding due to backwards incompatible change / missing functionality in the javax.measure.unit-api. 
I will stash this change in a separate commit so I can build the jar for the community and will remove it from the code.